dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -2032,6 +2035,10 @@ public void fireWorkflowPostCheckin(final WorkflowProcessor processor) throws Do
 					UtilMethods.isSet(processor.getContentlet().getIdentifier())) {
 
 				try {
+
+					processor.getContentlet().setProperty(Contentlet.WORKFLOW_IN_PROGRESS, Boolean.FALSE);
+					processor.getContentlet().setProperty(Contentlet.DO_REINDEX, Boolean.TRUE);
+
 					this.reindexQueueAPI.addIdentifierReindex(processor.getContentlet().getIdentifier());
 				} catch (DotDataException e) {
 					Logger.error(WorkflowAPIImpl.class, e.getMessage(), e);

```

Comment:
Please include in the error message the Identifier or useful information about the Contentlet involved in the process.